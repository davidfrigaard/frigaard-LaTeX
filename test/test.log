This is pdfTeX, Version 3.1415926-2.5-1.40.14 (TeX Live 2013/W32TeX) (format=pdflatex 2013.6.25)  5 SEP 2013 19:18
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**test.tex
(./test.tex
LaTeX2e <2011/06/27>
Babel <3.9f> and hyphenation patterns for 78 languages loaded.
(../frigaardreportF.cls

LaTeX Warning: You have requested document class `../frigaardreportF',
               but the document class provides `frigaardreportF'.

Document Class: frigaardreportF 2013/07/24
(c:/texlive/2013/texmf-dist/tex/latex/base/article.cls
Document Class: article 2007/10/19 v1.4h Standard LaTeX document class
(c:/texlive/2013/texmf-dist/tex/latex/base/size12.clo
File: size12.clo 2007/10/19 v1.4h Standard LaTeX file (size option)
)
\c@part=\count79
\c@section=\count80
\c@subsection=\count81
\c@subsubsection=\count82
\c@paragraph=\count83
\c@subparagraph=\count84
\c@figure=\count85
\c@table=\count86
\abovecaptionskip=\skip41
\belowcaptionskip=\skip42
\bibindent=\dimen102
)
(c:/texlive/2013/texmf-dist/tex/latex/etoolbox/etoolbox.sty
Package: etoolbox 2011/01/03 v2.1 e-TeX tools for LaTeX

(c:/texlive/2013/texmf-dist/tex/latex/etex-pkg/etex.sty
Package: etex 1998/03/26 v2.0 eTeX basic definition package (PEB)
\et@xins=\count87
)
\etb@tempcnta=\count88
)
(c:/texlive/2013/texmf-dist/tex/latex/psnfss/mathpazo.sty
Package: mathpazo 2005/04/12 PSNFSS-v9.2a Palatino w/ Pazo Math (D.Puga, WaS) 
\symupright=\mathgroup4
)
(c:/texlive/2013/texmf-dist/tex/latex/tex-gyre/tgcursor.sty
Package: tgcursor 2009/09/10 v1.2 TeX Gyre Cursor as default typewriter family

(c:/texlive/2013/texmf-dist/tex/latex/oberdiek/kvoptions.sty
Package: kvoptions 2011/06/30 v3.11 Key value format for package options (HO)

(c:/texlive/2013/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 1999/03/16 v1.13 key=value parser (DPC)
\KV@toks@=\toks14
)
(c:/texlive/2013/texmf-dist/tex/generic/oberdiek/ltxcmds.sty
Package: ltxcmds 2011/11/09 v1.22 LaTeX kernel commands for general use (HO)
)
(c:/texlive/2013/texmf-dist/tex/generic/oberdiek/kvsetkeys.sty
Package: kvsetkeys 2012/04/25 v1.16 Key value parser (HO)

(c:/texlive/2013/texmf-dist/tex/generic/oberdiek/infwarerr.sty
Package: infwarerr 2010/04/08 v1.3 Providing info/warning/error messages (HO)
)
(c:/texlive/2013/texmf-dist/tex/generic/oberdiek/etexcmds.sty
Package: etexcmds 2011/02/16 v1.5 Avoid name clashes with e-TeX commands (HO)

(c:/texlive/2013/texmf-dist/tex/generic/oberdiek/ifluatex.sty
Package: ifluatex 2010/03/01 v1.3 Provides the ifluatex switch (HO)
Package ifluatex Info: LuaTeX not detected.
)
Package etexcmds Info: Could not find \expanded.
(etexcmds)             That can mean that you are not using pdfTeX 1.50 or
(etexcmds)             that some package has redefined \expanded.
(etexcmds)             In the latter case, load this package earlier.
))))
(c:/texlive/2013/texmf-dist/tex/latex/base/fontenc.sty
Package: fontenc 2005/09/27 v1.99g Standard LaTeX package

(c:/texlive/2013/texmf-dist/tex/latex/base/t1enc.def
File: t1enc.def 2005/09/27 v1.99g Standard LaTeX file
LaTeX Font Info:    Redeclaring font encoding T1 on input line 43.
))
(c:/texlive/2013/texmf-dist/tex/latex/microtype/microtype.sty
Package: microtype 2013/05/23 v2.5a Micro-typographical refinements (RS)
\MT@toks=\toks15
\MT@count=\count89
LaTeX Info: Redefining \textls on input line 766.
\MT@outer@kern=\dimen103
LaTeX Info: Redefining \textmicrotypecontext on input line 1285.
\MT@listname@count=\count90

(c:/texlive/2013/texmf-dist/tex/latex/microtype/microtype-pdftex.def
File: microtype-pdftex.def 2013/05/23 v2.5a Definitions specific to pdftex (RS)

LaTeX Info: Redefining \lsstyle on input line 915.
LaTeX Info: Redefining \lslig on input line 915.
\MT@outer@space=\skip43
)
Package microtype Info: Loading configuration file microtype.cfg.

(c:/texlive/2013/texmf-dist/tex/latex/microtype/microtype.cfg
File: microtype.cfg 2013/05/23 v2.5a microtype main configuration file (RS)
))
(c:/texlive/2013/texmf-dist/tex/latex/fancyhdr/fancyhdr.sty
\fancy@headwidth=\skip44
\f@ncyO@elh=\skip45
\f@ncyO@erh=\skip46
\f@ncyO@olh=\skip47
\f@ncyO@orh=\skip48
\f@ncyO@elf=\skip49
\f@ncyO@erf=\skip50
\f@ncyO@olf=\skip51
\f@ncyO@orf=\skip52
)

Package Fancyhdr Warning: \fancyhead's `E' option without twoside option is use
less on input line 75.


Package Fancyhdr Warning: \fancyfoot's `E' option without twoside option is use
less on input line 76.

) (c:/texlive/2013/texmf-dist/tex/latex/lipsum/lipsum.sty
Package: lipsum 2011/04/14 v1.2 150 paragraphs of Lorem Ipsum dummy text
\c@lips@count=\count91
)
(c:/texlive/2013/texmf-dist/tex/latex/hyperref/hyperref.sty
Package: hyperref 2012/11/06 v6.83m Hypertext links for LaTeX

(c:/texlive/2013/texmf-dist/tex/generic/oberdiek/hobsub-hyperref.sty
Package: hobsub-hyperref 2012/05/28 v1.13 Bundle oberdiek, subset hyperref (HO)


(c:/texlive/2013/texmf-dist/tex/generic/oberdiek/hobsub-generic.sty
Package: hobsub-generic 2012/05/28 v1.13 Bundle oberdiek, subset generic (HO)
Package: hobsub 2012/05/28 v1.13 Construct package bundles (HO)
Package hobsub Info: Skipping package `infwarerr' (already loaded).
Package hobsub Info: Skipping package `ltxcmds' (already loaded).
Package hobsub Info: Skipping package `ifluatex' (already loaded).
Package: ifvtex 2010/03/01 v1.5 Detect VTeX and its facilities (HO)
Package ifvtex Info: VTeX not detected.
Package: intcalc 2007/09/27 v1.1 Expandable calculations with integers (HO)
Package: ifpdf 2011/01/30 v2.3 Provides the ifpdf switch (HO)
Package ifpdf Info: pdfTeX in PDF mode is detected.
Package hobsub Info: Skipping package `etexcmds' (already loaded).
Package hobsub Info: Skipping package `kvsetkeys' (already loaded).
Package: kvdefinekeys 2011/04/07 v1.3 Define keys (HO)
Package: pdftexcmds 2011/11/29 v0.20 Utility functions of pdfTeX for LuaTeX (HO
)
Package pdftexcmds Info: LuaTeX not detected.
Package pdftexcmds Info: \pdf@primitive is available.
Package pdftexcmds Info: \pdf@ifprimitive is available.
Package pdftexcmds Info: \pdfdraftmode found.
Package: pdfescape 2011/11/25 v1.13 Implements pdfTeX's escape features (HO)
Package: bigintcalc 2012/04/08 v1.3 Expandable calculations on big integers (HO
)
Package: bitset 2011/01/30 v1.1 Handle bit-vector datatype (HO)
Package: uniquecounter 2011/01/30 v1.2 Provide unlimited unique counter (HO)
)
Package hobsub Info: Skipping package `hobsub' (already loaded).
Package: letltxmacro 2010/09/02 v1.4 Let assignment for LaTeX macros (HO)
Package: hopatch 2012/05/28 v1.2 Wrapper for package hooks (HO)
Package: xcolor-patch 2011/01/30 xcolor patch
Package: atveryend 2011/06/30 v1.8 Hooks at the very end of document (HO)
Package: atbegshi 2011/10/05 v1.16 At begin shipout hook (HO)
Package: refcount 2011/10/16 v3.4 Data extraction from label references (HO)
Package: hycolor 2011/01/30 v1.7 Color options for hyperref/bookmark (HO)
)
(c:/texlive/2013/texmf-dist/tex/generic/ifxetex/ifxetex.sty
Package: ifxetex 2010/09/12 v0.6 Provides ifxetex conditional
)
(c:/texlive/2013/texmf-dist/tex/latex/oberdiek/auxhook.sty
Package: auxhook 2011/03/04 v1.3 Hooks for auxiliary files (HO)
)
\@linkdim=\dimen104
\Hy@linkcounter=\count92
\Hy@pagecounter=\count93

(c:/texlive/2013/texmf-dist/tex/latex/hyperref/pd1enc.def
File: pd1enc.def 2012/11/06 v6.83m Hyperref: PDFDocEncoding definition (HO)
)
\Hy@SavedSpaceFactor=\count94

(c:/texlive/2013/texmf-dist/tex/latex/latexconfig/hyperref.cfg
File: hyperref.cfg 2002/06/06 v1.2 hyperref configuration of TeXLive
)
Package hyperref Info: Hyper figures OFF on input line 4443.
Package hyperref Info: Link nesting OFF on input line 4448.
Package hyperref Info: Hyper index ON on input line 4451.
Package hyperref Info: Plain pages OFF on input line 4458.
Package hyperref Info: Backreferencing OFF on input line 4463.
Package hyperref Info: Implicit mode ON; LaTeX internals redefined.
Package hyperref Info: Bookmarks ON on input line 4688.
\c@Hy@tempcnt=\count95

(c:/texlive/2013/texmf-dist/tex/latex/url/url.sty
\Urlmuskip=\muskip10
Package: url 2006/04/12  ver 3.3  Verb mode for urls, etc.
)
LaTeX Info: Redefining \url on input line 5041.
\XeTeXLinkMargin=\dimen105
\Fld@menulength=\count96
\Field@Width=\dimen106
\Fld@charsize=\dimen107
Package hyperref Info: Hyper figures OFF on input line 6295.
Package hyperref Info: Link nesting OFF on input line 6300.
Package hyperref Info: Hyper index ON on input line 6303.
Package hyperref Info: backreferencing OFF on input line 6310.
Package hyperref Info: Link coloring OFF on input line 6315.
Package hyperref Info: Link coloring with OCG OFF on input line 6320.
Package hyperref Info: PDF/A mode OFF on input line 6325.
LaTeX Info: Redefining \ref on input line 6365.
LaTeX Info: Redefining \pageref on input line 6369.
\Hy@abspage=\count97
\c@Item=\count98
\c@Hfootnote=\count99
)

Package hyperref Message: Driver (autodetected): hpdftex.

(c:/texlive/2013/texmf-dist/tex/latex/hyperref/hpdftex.def
File: hpdftex.def 2012/11/06 v6.83m Hyperref driver for pdfTeX
\Fld@listcount=\count100
\c@bookmark@seq@number=\count101

(c:/texlive/2013/texmf-dist/tex/latex/oberdiek/rerunfilecheck.sty
Package: rerunfilecheck 2011/04/15 v1.7 Rerun checks for auxiliary files (HO)
Package uniquecounter Info: New unique counter `rerunfilecheck' on input line 2
82.
)
\Hy@SectionHShift=\skip53
) (./test.aux)
\openout1 = `test.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 15.
LaTeX Font Info:    ... okay on input line 15.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 15.
LaTeX Font Info:    ... okay on input line 15.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 15.
LaTeX Font Info:    ... okay on input line 15.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 15.
LaTeX Font Info:    ... okay on input line 15.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 15.
LaTeX Font Info:    ... okay on input line 15.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 15.
LaTeX Font Info:    ... okay on input line 15.
LaTeX Font Info:    Checking defaults for PD1/pdf/m/n on input line 15.
LaTeX Font Info:    ... okay on input line 15.
LaTeX Font Info:    Try loading font information for T1+pplj on input line 15.
 (c:/texlive/2013/texmf-dist/tex/latex/psnfss/t1pplj.fd
File: t1pplj.fd 2004/09/06 font definitions for T1/pplj.
)
\big@size=\dimen108
LaTeX Info: Redefining \microtypecontext on input line 15.
Package microtype Info: Generating PDF output.
Package microtype Info: Character protrusion enabled (level 2).
Package microtype Info: Using default protrusion set `alltext'.
Package microtype Info: Automatic font expansion enabled (level 2),
(microtype)             stretch: 20, shrink: 20, step: 1, non-selected.
Package microtype Info: Using default expansion set `basictext'.
Package microtype Info: No adjustment of tracking.
Package microtype Info: No adjustment of interword spacing.
Package microtype Info: No adjustment of character kerning.

(c:/texlive/2013/texmf-dist/tex/latex/microtype/mt-ppl.cfg
File: mt-ppl.cfg 2005/11/16 v1.6 microtype config. file: Palatino (RS)
)
\AtBeginShipoutBox=\box26
Package hyperref Info: Link coloring OFF on input line 15.

(c:/texlive/2013/texmf-dist/tex/latex/hyperref/nameref.sty
Package: nameref 2012/10/27 v2.43 Cross-referencing by name of section

(c:/texlive/2013/texmf-dist/tex/generic/oberdiek/gettitlestring.sty
Package: gettitlestring 2010/12/03 v1.4 Cleanup title references (HO)
)
\c@section@level=\count102
)
LaTeX Info: Redefining \ref on input line 15.
LaTeX Info: Redefining \pageref on input line 15.
LaTeX Info: Redefining \nameref on input line 15.

(./test.out) (./test.out)
\@outlinefile=\write3
\openout3 = `test.out'.

LaTeX Font Info:    Font shape `T1/pplj/m/sl' in size <14.4> not available
(Font)              Font shape `T1/pplj/m/it' tried instead on input line 17.
LaTeX Font Info:    Font shape `T1/pplj/m/sl' in size <17.28> not available
(Font)              Font shape `T1/pplj/m/it' tried instead on input line 17.
 [1

{c:/texlive/2013/texmf-var/fonts/map/pdftex/updmap/pdftex.map}] [1] [2]pdfTeX w
arning (ext4): destination with the same identifier (name{page.i}) has been alr
eady used, duplicate ignored
<to be read again> 
                   \relax 
l.23 \end
         {document} [1] [2]
Package atveryend Info: Empty hook `BeforeClearDocument' on input line 23.
 [3]
Package atveryend Info: Empty hook `AfterLastShipout' on input line 23.
 (./test.aux)
Package atveryend Info: Executing hook `AtVeryEndDocument' on input line 23.
Package atveryend Info: Executing hook `AtEndAfterFileList' on input line 23.
Package rerunfilecheck Info: File `test.out' has not changed.
(rerunfilecheck)             Checksum: D41D8CD98F00B204E9800998ECF8427E;0.
 ) 
Here is how much of TeX's memory you used:
 6706 strings out of 493315
 101237 string characters out of 6141455
 302080 words of memory out of 5000000
 9998 multiletter control sequences out of 15000+600000
 22698 words of font info for 110 fonts, out of 8000000 for 9000
 957 hyphenation exceptions out of 8191
 44i,11n,45p,770b,334s stack positions out of 5000i,500n,10000p,200000b,80000s
{c:/texlive/2013/texmf-dist/fonts/enc/dvips/base/8r.enc}<c:/texlive/2013/texm
f-dist/fonts/type1/public/fpl/fplrc8a.pfb><c:/texlive/2013/texmf-dist/fonts/typ
e1/public/fpl/fplrij8a.pfb><c:/texlive/2013/texmf-dist/fonts/type1/urw/palatino
/uplr8a.pfb><c:/texlive/2013/texmf-dist/fonts/type1/urw/palatino/uplri8a.pfb>
Output written on test.pdf (6 pages, 48201 bytes).
PDF statistics:
 49 PDF objects out of 1000 (max. 8388607)
 36 compressed objects within 1 object stream
 6 named destinations out of 1000 (max. 500000)
 19457 words of extra memory for PDF output out of 20736 (max. 10000000)

