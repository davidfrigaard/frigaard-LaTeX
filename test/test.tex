\documentclass[iltitle,iltoc,]{../frigaardreportF}

\author{David Frigaard}
\title{swag}
\date{\today}
%\cnum{00573043}
\email{mail@davidfrigaard.com}
\subtitle{subtutle subtitle}


\usepackage{lipsum}
\usepackage{hyperref}

\begin{document}
\pagelayout{pnomargin}
\maketitle
\tableofcontents
\section{Typography \& Typesetting}
The  classes use Zapf's Palatino as the default roman typeface. The sans-serif typeface of preference is \texttt{Bera Sans}. The typewriter typeface of choice is \texttt{Bera Mono}. Bera Sans was chosen over other popular sans-serif typefaces, such as Helvetica, because of its synergy with Bera Mono.\\

Micro-typography is served through the \texttt{microtype} package. 

\begin{fullwidth}
\rm How promotion excellent curiosity yet attempted happiness. Gay prosperous impression had conviction. For every delay death ask style. Me mean able my by in they. Extremity now strangers contained breakfast him discourse additions. Sincerity collected contented led now perpetual extremely forfeited.
\end{fullwidth}

\noindent Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Ut purus elit, vestibulum
ut, placerat ac, adipiscing vitae, felis. Curabitur dictum gravida mauris. Nam arcu
libero, nonummy eget, consectetuer id, vulputate a, magna.
\subsection{Implementation}
\lipsum[1]
\begin{verbatim}
\LaTeX\frigaard-LaTeX\documents
\end{verbatim}
\lipsum

\end{document}