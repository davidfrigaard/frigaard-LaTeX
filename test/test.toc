\select@language {english}
\contentsline {section}{\numberline {1}Typography \& Typesetting}{1}{section.1}
\contentsline {subsection}{\numberline {1.1}Implementation}{2}{subsection.1.1}
\contentsfinish 
