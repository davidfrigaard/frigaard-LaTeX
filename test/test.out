\BOOKMARK [1][-]{section.1}{Typography \046 Typesetting}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{Implementation}{section.1}% 2
